docs/tutorials/intro.md

```diff
@@ -0,0 +1,177 @@
+![Edit symbology](../assets/images/editSymbology.png)
+
+And when editing the symbology of the other Shapefile Layer:
+![Adjust Symbology](../assets/images/adjustSymbology.png)
+
+And updating the opacity (for instance to 0.5) of the US county Shapefile Layer, you can get a map that can look like to (depending on the colors and Stroke width you chose!):
+
+![final map for the intro exercise](../assets/images/finalIntroMap.png)
+```
+
+## Hiding and Reordering Layers
+
+### Hiding/Viewing Layers
+
+In the GIS Layers List/Browser Panel, you can select a layer and click the **eye** icon to hide or show the corresponding layer.
+![Hide a layer](../assets/images/hideLayer.png)
+
+### Reordering Layers
+
+The layers in your Layers List are displayed on the map according to their order in the list. The bottom layer is drawn first, while the top layer is drawn last. You can adjust their drawing order by rearranging the layers in the list.
+
+The order in which the layers have been loaded into the map is probably not logical at this stage. It’s possible that the road layer or/and the country layer are completely hidden because other layers are on top of it.
+
+For example, this layer order:
+![Wrong layer order](../assets/images/wrongLayerOrder.png)
+
+would hide all the custom shapefile layers.
+
+To solve this problem, you can click and drag on a layer in the Layers list to move it up or down, and reordering the layers as you wish them to be viewed on the map.
+
+## Renaming Layers
+
+As shown on the previous image, the two custom shapefile layers we added have the same name in the GIS Layers List / Browser Panel which can be very confusing.
+It is a good practice to rename your layers with meaningful names. To rename a layer, select it in the GIS Layers List / Browser Panel, and right click to **Rename Layer**.
+
+```{exercise-start}
+:label: Rename-layer
+```
+
+- Rename each **Custom Shapefile Layer** with a meaningful name e.g., **cb_2015_us_county_500k** for the US County Shapefile Layer and **nyc_roads** for the roads in New-York.
+
+```{exercise-end}
+
+```
+
+```{solution} Rename-layer
+:class: dropdown
+![Map with renamed layers](../assets/images/renameLayer.png)
+```
+
+## Save your JGIS project
+
+Your jGIS file corresponds to your GIS project (similar to `.qgz` for QGIS) and you probably already noticed that the default name is `untitled.jGIS`. If you created several jGIS files without renaming them, a number will be added before the extension e.g. `untitled1.jGIS`, `untitled2.jGIS`, etc.
+
+It is important to give meaningful name to your jGIS project.
+
+To rename it, you can click on **File** in the Jupyter toolbar menu and select **Rename JGIS...** to give a new name to your jGIS project.
+
+![Rename jGIS project](../assets/images/renamejGIS.png)
+
+## Conclusion
+
+Now you’ve learned how to add all the layers you need from various sources and have created a basic map!
```

Comment:
We could go into a bit more depth here (repeating some content from the top of the tutorial), and possibly provide some more positivity! What about something along the lines of:


```suggestion
Congratulations on completing this tutorial! You've created a basic map in JupyterGIS, learned how to add new layers, rearrange layers, alter symbology, and save your project. You're now well on your way to becoming a pro JupyterGIS user!
```